Ginger/Ginger/WizardLib/WizardWindow.xaml.cs

```diff
@@ -254,26 +254,25 @@ public void SearchValidationsRecursive(DependencyObject depObj)
 
         private void UpdatePrevNextButton()
         {
-
-            if (mWizard.IsLastPage())
+            if (mWizard.IsFirstPage())
             {
-                xNextButton.IsEnabled = false;
-                xFinishButton.IsEnabled = true;
+                xPrevButton.IsEnabled = false;
```

Comment:
to eliminate risk of impacting other wizards, please roll back to original code and only add to first if condtion the below:
if (mWizard.IsLastPage())
            {
                xNextButton.IsEnabled = false;
                xFinishButton.IsEnabled = true;
	-->> Add: If (mWizard.DisableBackBtnOnLastPage == true)
		{
			xPrevButton.IsEnabled = false;
		}
            }
            else
	...

Reply:
Yes Meni, Done with requested changes. This PR took long time, sorry for that.

Reply:
but i don't see you rolled back to original code.
this UpdatePrevNextButton() function still includes many changes.